core/proto/core.proto

```diff
@@ -22,6 +22,7 @@ message ValidationRecord {
 
         repeated string authorities = 5;
         optional string url = 6;
+        repeated bytes addressesTried = 7; // net.IP.MarshalText()
```

Comment:
Can you add a comment describing the semantics of this record? I.e., is it expected to contain `addressUsed`? Is there ordering?